source/app/service-providers/latex-commands.ts

```diff
@@ -0,0 +1,40 @@
+import { promises as fs } from 'fs'
+import defaultCommands from '../../../static/latex-commands.json'
+
+export interface LatexCommand {
+  name: string
+  displayText: string
+  snippet: string
+  description?: string
+  definition?: string
+}
+
+function convert (command: {
```

Comment:
Do I read it correctly that this basically just makes sure that the commands are usable? If so, `convert` may be a tad misleading …?

Reply:
It's "converting" the json representation to the `LatexCommands` object (which happen to be almost the same thing). Do you have a suggestion for a better name?